tests/functional/workloads/cnv/test_vm_lifecycle_and_io.py

```diff
@@ -33,6 +34,8 @@ def test_vm_lifecycle_and_io(self, cnv_workload, setup_cnv):
         6) Delete the VM (as part of factory teardown)
 
         """
+        file_path = "/io_tests"
+        fio_service_name = "fio_test"
```

Comment:
Can this be handled in the run_fio function by generating unique names, so we don't need to pass them in the test?

Reply:
Take care in run_io function parameters.
https://github.com/red-hat-storage/ocs-ci/pull/11624/files#diff-9ef67c77e288e54e4755c5038e0fb35247a50df534ad1c4f44b4914c0ca616e2R531